openfecwebapp/templates/legal-advisory-opinion-landing.html

```diff
@@ -0,0 +1,53 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal.html' as legal %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources')] %}
+
+{% block title %}Search advisory opinions{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  {{ breadcrumb.breadcrumbs('Advisory opinions', breadcrumb_links) }}
+</header>
+
+<section class="main__content--full data-container__wrapper">
+  {{ legal.filter('Search advisory opinions', 'advisory_opinions') }}
+  <div class="content__section data-container__body">
```

Comment:
I just noticed that in my refactor of the filter panel / data-container layout, I removed the left padding on `.data-container__body`, which works for when it's actually a data table, but means that in this layout, the content runs up to the edge of the screen.